### PR TITLE
Update crash-course.md

### PR DESCRIPTION
**Pull request by mansuleman**

### CODE REVIEW

**Comment by mansuleman** [comment]
few small edits for now. Right pull request?

File: basics/crash-course.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
 ## Crash Course to CompoundJS
 
-In this guide we will start with creating app using generators. Goal: get
-working application without any additional knoweledge about express, learn
-structure and tools coming with railway, quick overview of main features.
+The guide demonstrates how to creating simple compound app using generators.
```
Comment (1602) [inline basics/crash-course.md:3]: Maybe "how to create"?

**Comment by 1602** [comment]
This is how this pull request should look like: https://github.com/compoundjs/guides/pull/8

**Comment by mansuleman** [comment]
Is: #7 OK as a pull request?

**Comment by 1602** [comment]
it's fine

On Mon, Mar 18, 2013 at 10:42 AM, Mansur S notifications@github.com wrote:

> Is: #7 https://github.com/compoundjs/guides/issues/7 OK as a pull
> request?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/compoundjs/guides/pull/2#issuecomment-15041214
> .

## 

Thanks,
Anatoliy Chakkaev

**Pull request closed without merging**